presto-main/src/main/java/com/facebook/presto/execution/SharedBuffer.java

```diff
@@ -149,13 +149,19 @@ public boolean isTerminal()
 
     public SharedBuffer(TaskId taskId, Executor executor, DataSize maxBufferSize)
     {
+        this(taskId, executor, maxBufferSize, e -> { });
```

Comment:
don't use single letter variable names, so name this `deltaMemory`. Also, I don't think we put a space between empty `{}`